integration/windows/Dockerfile.DeploymentApiMock

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.12
+WORKDIR c:/ws/
+RUN git clone --branch v0.2.0 https://github.com/Dynatrace/deployment-api-mock.git
```

Comment:
do we really want to have that specific branch here?

Reply:
Git is calling it a branch, but in this case I'm using it to refer to the v0.2 tag/release. I'm setting versions explicitly to avoid breaking things if I do a major change on the library.